inferno-ml-server-types/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Revision History for inferno-ml-server-types
+*Note*: we use https://pvp.haskell.org/ (MAJOR.MAJOR.MINOR.PATCH)
+
+## 0.0.1
+* Initial pre-release
```

Comment:
I would prefer to include a textual tag indicating a pre-release, but Cabal doesn't like that